### PR TITLE
Fix gpu and default params

### PR DESCRIPTION
**Pull request by lscheinkman**
Added missing default values for the `experiments_paper.cfg` and fixed `daemonic processes are not allowed to have children` error when using GPU.

@subutai I was able to run all the 8 experiments in experiments_paper.cfg in 20 minutes after this fix using domino's 
`GPU g3.4xlarge (CUDA 9)` machine combined with `htmresearch dev (GPU Tools CUDA9)`

### CODE REVIEW

**Comment by subutai** [comment]
Excellent!

File: projects/sdr_paper/pytorch_experiments/experiments_paper.cfg [hunk 1]
```diff
@@ -21,6 +21,9 @@ create_plots = False
 count_nonzeros = True
 test_noise_every_epoch = True
 
+use_batch_norm = True
```
Comment (subutai) [inline projects/sdr_paper/pytorch_experiments/experiments_paper.cfg]: should we default to False for MNIST since batch norm didn't help?

Comment (lscheinkman) [reply projects/sdr_paper/pytorch_experiments/experiments_paper.cfg]: yes
